packages/varnish/src/components/VarnishApp.tsx

```diff
@@ -23,7 +24,17 @@ export interface VarnishAppProps {
  * and those not provided will use the defaults.
  */
 export function VarnishApp({ layout, theme, children }: VarnishAppProps) {
-  const mergedTheme = getTheme(theme);
+
+  const mergedTheme = createTheme(deepmerge(getTheme(theme), {
```

Comment:
this does not belong here, rather it should go into the theme:
https://github.com/allenai/varnish-mui/blob/3182b2ad6e9476a3b0da1e7256408b8b7f9e47f6/packages/varnish/src/theme/theme.ts#L62

Reply:
fixed! thanks for the callout.